clang-tools-extra/clangd/IncludeCleaner.h

```diff
@@ -62,15 +64,6 @@ issueIncludeCleanerDiagnostics(ParsedAST &AST, llvm::StringRef Code,
                                const ThreadsafeFS &TFS,
                                HeaderFilter IgnoreHeader = {});
 
-/// Affects whether standard library includes should be considered for
-/// removal. This is off by default for now due to implementation limitations:
-/// - macros are not tracked
-/// - symbol names without a unique associated header are not tracked
-/// - references to std-namespaced C types are not properly tracked:
-///   instead of std::size_t -> <cstddef> we see ::size_t -> <stddef.h>
-/// FIXME: remove this hack once the implementation is good enough.
-void setIncludeCleanerAnalyzesStdlib(bool B);
```

Comment:
This function was removed in e289ee99cec4607243aeaa01504f6b3cf65b65fe, but the declaration was left for some reason (overlooked?)

Reply:
At first I considered adding a global flag similar to `--include-cleaner-stdlib`, but then thought that having more fine-grained control would be better.

Reply:
completely agreed, thanks for cleaning this up!